build.sbt

```diff
@@ -4,8 +4,8 @@
 
 lazy val `routeguide-protocol` = project
   .in(file("routeguide/protocol"))
-  .settings(libraryDependencies ++= Seq(mu("mu-rpc-monix")))
-  .disablePlugins(SrcGenPlugin)
+  .settings(libraryDependencies ++= Seq(mu("mu-rpc-monix"), mu("mu-rpc-fs2")))
```

Comment:
Do we need `mu-rpc-fs2` in this module if we are using monix observables?

Reply:
oooh yeah we do not.  Good eye.